tests/test_configs/smartredis/consumer.py

```diff
@@ -41,7 +41,7 @@
     args = parser.parse_args()
 
     # get model and set into database
-    #NOTEjp:  c = Client(False,db_identifier) retrieving the db_identfier somehow 
+    # NOTEjp:  c = Client(False,db_identifier) retrieving the db_identfier somehow
```

Comment:
can we actually just drop this comment? I don't think it is needed anymore

Reply:
Done